From 19f561b36be254801ffc6cb6a4f57874c6544d64 Mon Sep 17 00:00:00 2001
From: Hangbin Liu <liuhangbin@gmail.com>
Date: Tue, 26 Feb 2013 10:03:59 +0800
Subject: [PATCH 48/53] test_1_to_1_threads: remove unused variable for t_recv
 and t_send

I think this is enough, we can't give NULL to pthread_create as we need to
make sure there is one thread(I use the first one) only run t_send in case of
test huang for ever.

Signed-off-by: Hangbin Liu <liuhangbin@gmail.com>
Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 src/func_tests/test_1_to_1_threads.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/func_tests/test_1_to_1_threads.c b/src/func_tests/test_1_to_1_threads.c
index 562b4d8..e2a7eee 100644
--- a/src/func_tests/test_1_to_1_threads.c
+++ b/src/func_tests/test_1_to_1_threads.c
@@ -64,7 +64,7 @@ struct sockaddr_in  conn_addr;
 char *message = "hello, world!\n";
 
 void 
-t_recv (int id) {
+t_recv(void) {
 	int cnt;
 	struct msghdr inmessage;
 	struct iovec iov;
@@ -87,7 +87,7 @@ t_recv (int id) {
 }
 
 void
-t_send(int id) {
+t_send(void) {
         struct msghdr outmessage;
         struct sctp_sndrcvinfo *sinfo;
         struct cmsghdr *cmsg;
@@ -122,10 +122,10 @@ void *relay(void *arg)
 	int id = *(int *) arg;
 
 	if (id == 0) {
-		t_send(id);
+		t_send();
 	} else {
-		t_recv (id);
-		t_send(id);
+		t_recv();
+		t_send();
 	}
 
 	pthread_exit(NULL);
-- 
1.7.11.7

